static/js/src/advantage/users/components/AddNewUser/AddNewUserForm.tsx

```diff
@@ -97,8 +97,8 @@ export const AddNewUserForm = ({
                 or support portal.
```

Comment:
Also remove this line?

Reply:
well it says they can't get to it.. so it's also true for now :)